context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+ContextFree is for people who want to create designs with lots of similar 
+geometric images that rely on symmetry.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to create symmetrical geometric patterns concisely because 
```

Comment:
Saying that creating recursive patterns is easy is maybe more correct than symmetrical, but I wouldn't say that saying it's good at making symmetrical geometric patterns is wrong (it's really good at it, in fact!).